<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="17737" started-at="2019-02-09T21:46:40Z" finished-at="2019-02-09T21:46:58Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="17737" started-at="2019-02-09T21:46:40Z" finished-at="2019-02-09T21:46:58Z">
      <class name="DemoEnvDif2EventReportingpage">
        <test-method status="PASS" signature="initialization()[pri:0, instance:DemoEnvDif2EventReportingpage@735b5592]" name="initialization" is-config="true" duration-ms="17705" started-at="2019-02-09T21:46:40Z" finished-at="2019-02-09T21:46:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method status="FAIL" signature="DIF1_FilloutNonLogin()[pri:0, instance:DemoEnvDif2EventReportingpage@735b5592]" name="DIF1_FilloutNonLogin" duration-ms="9" started-at="2019-02-09T21:46:58Z" finished-at="2019-02-09T21:46:58Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at DemoEnvDif2EventReportingpage.DIF1_FilloutNonLogin(DemoEnvDif2EventReportingpage.java:75)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DIF1_FilloutNonLogin -->
      </class> <!-- DemoEnvDif2EventReportingpage -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
